src/js/form.js

```diff
@@ -27,6 +28,85 @@ const schema = {
       }
     },
     pronouns: { type: "string", title: "Pronouns" },
+    skills: {
+      title: "My Skills",
+      type: "array",
+      uniqueItems: true,
+      items: {
+        type: "string",
+        enum: [
+          "AI",
+          "Data Mining",
+          "NLP",
+          "Web Development",
+          "IOS",
+          "Android",
+          "Pitching",
+          "Marketing",
+          "Design",
+          "AR/VR",
+          "Game Development",
+          "Systems"
+        ]
+      }
+    },
+    commitment: {
+      title: "Commitment Level",
+      type: "string",
+      enum: [
+        "Low",
+        "Medium",
+        "High"
+      ]
+    },
+    timezoneOffset: {
+      title: "Timezone",
+      type: "string",
+      enum: [
+        "UTC−12:00",
```

Comment:
I don't know if people will easily know which timezone offset they are in. Maybe you can either 1) show locations next to each time zone (such as "UTC-... New York" or 2) [detect the user's timezone](https://stackoverflow.com/questions/1091372/getting-the-clients-timezone-offset-in-javascript) to pre-fill this question, or both of the above.

Reply:
Configured it so it sets GMT automatically for a first-time user